src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseScanTests/Licenses.source-build-externals.json

```diff
@@ -2,7 +2,11 @@
   "files": [
     {
       "path": "src/application-insights/LOGGING/ThirdPartyNotices.txt",
-      "detected_license_expression": "unknown AND apache-2.0 AND mit AND bsd-new"
+      "detected_license_expression": "(unknown-license-reference AND apache-2.0) AND mit AND bsd-new"
+    },
+    {
+      "path": "src/application-insights/WEB/ThirdPartyNotices.txt",
+      "detected_license_expression": "(unknown-license-reference AND bsd-new) AND mit AND ms-pl AND apache-2.0 AND (cc-by-3.0-us AND cc-by-3.0 AND mit) AND ms-rl"
     }
```

Comment:
This should be removed. The `unknown-license-reference` should be defined as a false positive in the exclusions file.